components/de_amp/browser/test/BUILD.gn

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import("//testing/test.gni")
+
+source_set("unit_tests") {
+  testonly = true
+  sources = [ "de_amp_util_unittest.cc" ]
+  deps = [
+    "///brave/components/de_amp/browser",
+    "//base/test:test_support",
+    "//components/prefs:test_support",
+  ]
+  defines = [ "HAS_OUT_OF_PROC_TEST_RUNNER" ]
+}  # source_set("unit_tests")
```

Comment:
I'm not sure why we have these `# source_set(` comments in some places, it's not a convention used in upstream and isn't really useful for such a short block anyway